apps/c/adi_burger/adi_burger.cpp

```diff
@@ -229,5 +229,6 @@ int main(int argc, const char **argv) {
     ops_fetch_dat_hdf5_file(v, "adi_burger2D.h5");
 
     ops_printf("\nTotal Wall time %lf\n", et1 - et0);
+    ops_printf("\nSuccessful exit from OPS!\n");
```

Comment:
Why have these? And why do they happen before the actual exit? While we are at it, please insert return 0 after ops_exit.

Reply:
done